src/plugins/data/public/ui/filter_bar/filter_editor/value_input_type.tsx

```diff
@@ -147,6 +147,17 @@ class ValueInputTypeUI extends Component<Props> {
     this.props.onChange(boolValue);
   };
 
+  private onNumberChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const params = event.target.value;
+    let numValue;
+    if (typeof params === 'string') {
+      numValue = parseFloat(params);
+    } else if (typeof params === 'bigint') {
+      numValue = (params as BigInt).toString();
```

Comment:
Where are we using `numValue`?

Reply:
should be `this.props.onChange(numValue);`. It is a typo when addressing later comments

Reply:
will fix soon